tracks/bash/exercises/pangram/mentoring.md

```diff
@@ -0,0 +1,42 @@
+## Common Suggestions
+
+* use brace expansion
+
+```md
+On LN, consider a more concise way to provide a sequence of letters.
+[Brace Expansion](https://www.gnu.org/software/bash/manual/html_node/Brace-Expansion.html)
+in the Bash Reference Manual might be useful.
+```
+
+## General Guidelines
+
+* there is no `main` function
+
+```md
+Generally, you should encapsulate the main body of your script in a `main`
+function, like you may remember from the Hello World exercise.  It
```

Comment:
The "Hello World" is a title, we can use title indicators for this, such as one would use for a book or an article.

Perhaps:

_Hello World_ to italicize it?